SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/OrderController.cs

```diff
@@ -6,8 +6,29 @@
 using System.Net;
 using System.Web;
 using System.Web.Mvc;
-using SeaSharpe_CVGS.Models;
+using SeaSharpe_CVGS.Models;
 
+/*
+ * This Controller tracks orders for site members and employees.
+ * 
+ * **************For Members*********************
+ * Cart:
+ * can add items to a cart
+ * view their cart
+ * delete from their cart
+ * purchase games from their cart
+ * 
+ * OrderHistory
+ * View ordered games and the ship dates
+ * 
+ * **************For Employees*******************
+ * Order Management:
+ * View all completed orders
+ * with partial views for orders waiting to be processed and a partial view to view the selected order
+ * which also provides options to mark the selected order as "processed" as well as print a shipping 
+ * label with the pertinent details.
+ */

```

Comment:
Also they should be in this format so that they can be picked up by intellisense and the documentation generating script: 
```csharp
/// <summary>
/// These are valid doc comments
/// </summary>
public class OrderController : Controller
{
	/// <summary>
	/// Buy's postback action
	/// </summary>
	/// <param name="stripeToken">
	///     The token provided to the client to track this purchase
	/// </param>
	/// <param name="stripeTokenType">The type of token, usualy "card"</param>
	/// <param name="stripeEmail">
	///     The email the customer provided in the stripe payment dialogue box
	/// </param>
	/// <returns>
	///     Returns a view displaying the status of the payment to the user.
	/// </returns>
	[HttpPost]
	public ActionResult Buy(string stripeToken, string stripeTokenType, string stripeEmail)
	{
		var chargeOptions = new StripeChargeCreateOptions
		{
			Amount = 5153,
			Currency = "CAD",
			Description = "Charge it like it's hot",
			Source = new StripeSourceOptions
			{
				Object = stripeTokenType,
				TokenId = stripeToken,
				ReceiptEmail = stripeEmail
			}
		};
		Buy()
		var chargeService = new StripeChargeService();
		var stripeCharge = chargeService.Create(chargeOptions);
		return View(stripeCharge);
	}
}
```

![intelesense example](https://cloud.githubusercontent.com/assets/3138467/11605635/1b95a7c0-9ad3-11e5-80ed-0f77842162c3.png)